client/components/Highlight/Highlight.module.scss

```diff
@@ -2,70 +2,8 @@
 
 .highlight {
   width: 100%;
-  margin-left: auto;
```

Comment:
In the end we want to have both the `Highlight` component and the `IFrame` components. I know you deleted this code just to quickly test the ifame scenario :)